packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -0,0 +1,269 @@
+import ExpoAppleAuthentication from './ExpoAppleAuthentication';
+import { StyleProp, ViewStyle } from 'react-native';
+
+export interface ButtonProps {
+  /**
+   * The callback which is called when the user pressed the button.
+   */
+  onPress: () => void;
+
+  /**
+   * Controls the text that is shown on the button.
+   */
+  buttonType: ButtonType;
+
+  /**
+   * Controls the style of the button.
+   */
+  buttonStyle: ButtonStyle;
+
+  /**
+   * The radius of the corners of the button.
+   */
+  cornerRadius?: number;
+
+  style?: StyleProp<ViewStyle>;
+}
+
+/**
+* The options you can supply when making a call to `AppleAuthentication.requestAsync()`.
+*
+* @see [Apple Documentation](https://developer.apple.com/documentation/authenticationservices/asauthorizationopenidrequest) for more details.
+*/
+export interface RequestOptions {
+  /**
+    * The scopes that you are requesting.
```

Comment:
This is another comment that doesn't add marginal info -- remove or explain the non-obvious behaviors.